### ISSUE TITLE
referencing gem via git triggers missing bootstrap dependency

### ISSUE THREAD

**Issue by jcoyne** [voormedia/flipflop#10]
If you put this in your Gemfile:

``` ruby
gem 'flipflop', git: 'https://github.com/voormedia/flipflop.git'
```

And you don't include bootstrap 4, then you will get an error like:

```
Sass::SyntaxError at /

File to import not found or unreadable: bootstrap/reboot.
```

**Comment by jcoyne** [comment]
Perhaps moving `flipflop.scss` out of `app/assets/stylesheets/` would resolve this?

**Comment by rolftimmermans** [comment]
Fixed in 2f7860e.
